mu-plugins/blocks/time/src/convert_times.js

```diff
@@ -0,0 +1,80 @@
+function convertTimes() {
```

Comment:
File names should be separated with a hyphen, so this should be `convert-times.js`. Or you could use `view.js` if this is meant to be the `viewScript`— I don't have a strong opinion about following that convention though.

Reply:
Renamed